File: julia/mat_solve_1.jl
#!/usr/bin/env julia
#using LinearAlgebra

# 3 * x + y = 9
# x + 2 * y = 8
a = [3 1 ; 1 2]
b = [9 ; 8]
ai = inv(a)
s = ai * b

print(s, "\n")

File: py/numpy/mat_solve_1.py
#!/usr/bin/env python3
import numpy as np

# 3 * x + y = 9
# x + 2 * y = 8
a = np.mat([[3, 1], [1, 2]])
b = np.array([9, 8])
s = np.dot(a.I, b)
# a.I == a.getI() == np.linalg.inv(a)
print(s)

